src/transformed/transformed-light.json

```diff
@@ -427,32 +427,39 @@
     "disabled": {
       "text": {
         "value": "#596D84",
-        "type": "color"
+        "type": "color",
+        "description": "For use in disabled content like text and icons"
       },
       "background": {
         "value": "#F5F7FA",
-        "type": "color"
+        "type": "color",
+        "description": "Used as the background colour for disabled components"
       },
       "nakedContent": {
         "value": "#F5F7FA",
-        "type": "color"
+        "type": "color",
+        "description": "Used for content in disabled components with little to no styling"
```

Comment:
I'm sure there's a better way to describe nakedContent other than "little or no styling". We should also re-evaluate the need for this token, it's only being used in the Pill as a background color: https://git.fullscript.io/developers/hw-admin/-/blob/staging/app/javascript/aviary/components/Pill/Pill.styles.ts#L79

Reply:
I think that makes sense!

Reply:
good catch - maybe we can decide if we need this token during our meeting today?

Reply:
I've gone ahead and removed the desc from nakedContent, will open a follow up MR to remove it completely